pynumdiff/total_variation_regularization/_chartrand_tvregdiff.py

```diff
@@ -145,84 +144,50 @@
 #########################################################
 
 import sys
-
-try:
-    import numpy as np
-    import scipy as sp
-    from scipy import sparse
-    from scipy.sparse import linalg as splin
-except ImportError:
-    sys.exit("Numpy and Scipy must be installed for TVRegDiag to work - "
-             "aborting")
-
-_has_matplotlib = True
-
-try:
-    import matplotlib.pyplot as plt
-except ImportError:
-    _has_matplotlib = False
-    print("Matplotlib is not installed - plotting functionality disabled")
```

Comment:
I don't think it's worth having all this up here. If you try to import something but don't have it, python will give you a message about it already.